### ISSUE TITLE
Translations passed as nested attributes; nil entry on create, fine on edit

### ISSUE THREAD

**Issue by lloydmeta** [globalize/globalize#148]
TL:DR: Is there a way to skip the creation of the translation object that is created based on the current I18n and the on-object attributes?

On the Tokyorails project (https://github.com/tokyorails/tokyorails), I've implemented localisation of the Events and Projects class using Globalize3 and I really liked how easy seamless it was.

That said, I'm currently running into an issue with trying to create a localisable object (e.g. Project that :has_many ProjectTranslations) in ActiveAdmin via the ActiveAdmin-Globalize3-Inputs gem (just creates Jquery-UI tabs). In terms of appearance, it looks great and it looks like this : http://img31.imageshack.us/img31/1554/screenshot20120629at811.png

On the Project model side, I've added :

``` ruby
accepts_nested_attributes_for :project_translations, :allow_destroy => true 
```

as well as a ProjectTranslation class that just has a ':belongs_to projects' line. 

Everything works fine except on creation, where a translation for the current I18n.locale is created in addition to the one passed in as an attribute. To illustrate: http://img11.imageshack.us/img11/3950/screenshot20120629at812.png  . Notice that there are 2 project_translations_attributes passed in via the hash, but there are 3 SQL statements for creating project_translations:

```
Started POST "/admin/projects?locale=en" for 127.0.0.1 at 2012-06-29 08:12:00 +0900
Processing by Admin::ProjectsController#create as HTML
Parameters: {"utf8"=>"✓", "authenticity_token"=>"+RZLytYxCDUU16U587MR4Sx0Jk/m3Fxgaxg1svOwIrY=", "project"=>{"member_id"=>"45", "github_url"=>"https://github.com/tokyorails/tokyorails", "photo_url"=>"http://www.tokyorails.org/assets/tokyo-rails-400x193.png", "project_translations_attributes"=>{"0"=>{"title"=>"My Project Title", "description"=>"", "html"=>"", "locale"=>"en"}, "1"=>{"title"=>"", "description"=>"", "html"=>"", "locale"=>"ja"}}}, "commit"=>"Create Project", "locale"=>"en"}
```

...

``` sql
SQL (4.3ms)  INSERT INTO "project_translations" ("created_at", "description", "html", "locale", "project_id", "title", "updated_at") VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING "id"  [["created_at", Fri, 29 Jun 2012 08:12:00 JST +09:00], ["description", ""], ["html", ""], ["locale", "en"], ["project_id", 27], ["title", "My Project Title"], ["updated_at", Fri, 29 Jun 2012 08:12:00 JST +09:00]]
SQL (0.7ms)  INSERT INTO "project_translations" ("created_at", "description", "html", "locale", "project_id", "title", "updated_at") VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING "id"  [["created_at", Fri, 29 Jun 2012 08:12:00 JST +09:00], ["description", ""], ["html", ""], ["locale", "ja"], ["project_id", 27], ["title", ""], ["updated_at", Fri, 29 Jun 2012 08:12:00 JST +09:00]]
SQL (0.5ms)  INSERT INTO "project_translations" ("created_at", "description", "html", "locale", "project_id", "title", "updated_at") VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING "id"  [["created_at", Fri, 29 Jun 2012 08:12:00 JST +09:00], ["description", nil], ["html", nil], ["locale", "en"], ["project_id", 27], ["title", nil], ["updated_at", Fri, 29 Jun 2012 08:12:00 JST +09:00]]
```

On edit, this doesn't happen.

Currently, my fix is to override the 'create' action on the ActiveAdmin generated controller and delete the ProjectTranslation object with all nil attributes for the just saved Project, but this is kind of ugly. I think one reason this is happening is that on the create form, I don't actually set the translated attributes (e.g. title, description) of project directly; they're all created as part of the project_translations_attributes (looking at the parameters, these are nil).

So, my question is: Is there a way to skip the creation of the translation object that is created based on the current I18n and the on-object attributes?

Sorry for the long-winded question/writeup.
